Add "Known issue" in "Develop an extension for the Microsoft Edge sidebar" (#3328)
MicrosoftDocs/edge-developer: microsoft-edge/extensions-chromium/developer-guide/sidebar.md

![The sidebar for a Microsoft Edge extension](./sidebar-images/sidebar-screenshot.png)

> The sidebar extensions feature is being rolled out to a growing user base for all channels of Microsoft Edge.<!-- todo: after rollout is complete, delete sentence -->

Reviewer: feature is being rolled out

Reviewer: i deleted "The sidebar extensions feature is being rolled out to a growing user base for all channels of Microsoft Edge."
@bhuvanapriyap @captainbrosset

Features of the Sidebar API include:

> * The sidebar remains open while navigating between tabs.  Known issue: The sidebar is not automatically displayed again when the user switches to a tab in which the sidebar was previously open ([Issue #142](https://github.com/microsoft/MicrosoftEdge-Extensions/issues/142)).<!-- todo: update -->

Reviewer: is there a faq to also link to?
2x
@bhuvanapriyap

Author: no @mikehoffms , there is no faq as of now

Features of the Sidebar API include:

> * The sidebar remains open while navigating between tabs.<!-- todo: fix/reconcile -->

Reviewer: is there an Issue to link to?

went live July 7, 2023:
[Sidebar API for extensions is now live! · microsoft/MicrosoftEdge-Extensions · Discussion #95](https://github.com/microsoft/MicrosoftEdge-Extensions/discussions/95) 
 
Search for Issues w/ "tab":
https://github.com/microsoft/MicrosoftEdge-Extensions/discussions?discussions_q=is%3Aopen+tab
possible contact in apparently related Issue: 
[Cannot remove/update NTP tab issue · microsoft/MicrosoftEdge-Extensions · Discussion #217](https://github.com/microsoft/MicrosoftEdge-Extensions/discussions/217)

https://github.com/ManikanthMSFT
@ManikanthMSFT

Reviewer 2: I think this is the issue we should link to: https://github.com/microsoft/MicrosoftEdge-Extensions/issues/142 from your note below.

>   Note: Currently, the sidebar is not automatically displayed again when the user switches to a tab in which the sidebar was previously open.<!-- todo: if behavior changes, update or remove statement -->

Reviewer 2: Instead of "Note:", what about using "Known issue:"?

```suggestion
  Known issue: the sidebar is not automatically displayed again when the user switches to a tab in which the sidebar was previously open, see the [tracking bug on the microsoft/MicrosoftEdge-Extensions repo](https://github.com/microsoft/MicrosoftEdge-Extensions/issues/142).<!-- todo: if behavior changes, update or remove statement -->
```

Reviewer: thx, spectrum of options, from minor to heavy:
* "Note:" text
* `> [!NOTE]` alert ([depr.](https://review.learn.microsoft.com/en-us/help/platform/markdown-reference?branch=main#alerts-note-tip-important-caution-warning)) 
* "Known issue:" text
* "Known issue" minimal heading
* "Known issue: Foo is bar" detailed heading

Reviewer: result 2x: 
```
Known issue: The sidebar is not automatically displayed again when the user 
switches to a tab in which the sidebar was previously open; see 
[Issue 142](https://github.com/microsoft/MicrosoftEdge-Extensions/issues/142).
<!-- todo: update -->
```

When a user switches to a tab or site for which the sidebar is not enabled, the sidebar is hidden.

> Note: Currently, sidebar will not automatically show again when the user switches to a tab where it was previously open.

Reviewer: ```suggestion
The sidebar is not automatically displayed again when the user switches to a tab in which the sidebar was previously open.
```

> The sidebar is not automatically displayed again when the user switches to a tab in which the sidebar was previously open.<!-- todo: if behavior changes, update or remove statement -->

Reviewer: Is there a way for the extension developer to persist the display of the sidebar in the tab?

rendered section:
https://review.learn.microsoft.com/en-us/microsoft-edge/extensions-chromium/developer-guide/sidebar?branch=pr-en-us-3328#enable-a-sidebar-for-a-specific-site-only

Reviewer: status:
https://github.com/microsoft/MicrosoftEdge-Extensions/issues/142

When a user switches to a tab or site for which the sidebar is not enabled, the sidebar is hidden.

> Note: Currently, the sidebar is not automatically displayed again when the user switches to a tab in which the sidebar was previously open.<!-- todo: if behavior changes, update or remove statement -->

Reviewer: should we have same note here as well as above?  

rendered article section:
https://review.learn.microsoft.com/en-us/microsoft-edge/extensions-chromium/developer-guide/sidebar?branch=pr-en-us-3328#enable-a-sidebar-for-a-specific-site-only - below the code listing, find "not automatically displayed"

Reviewer 2: This is a good idea.

<!-- ====================================================================== -->
> [!NOTE]
> Portions of this page are modifications based on work created and [shared by Google](https://developers.google.com/terms/site-policies) and used according to terms described in the [Creative Commons Attribution 4.0 International License](https://creativecommons.org/licenses/by/4.0).

> > The original page is found [here](https://developer.chrome.com/docs/extensions/reference/api/sidePanel).

Reviewer: pls take a look at updated upstream article:
https://developer.chrome.com/docs/extensions/reference/api/sidePanel
@bhuvanapriyap 

rendered article in PR:
https://review.learn.microsoft.com/en-us/microsoft-edge/extensions-chromium/developer-guide/sidebar?branch=pr-en-us-3328

Reviewer 2: Seems good to me, the old URL now redirects to the new one.
